apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
:confused: That's autogenerated.  I thought the whole point of yarn.lock was to lock versions? I wonder if it's new yarn behavior after the recent upgrade - I don't see "latest" many other places in that file.

To be fair, it looks like the versions of those dependencies are still "locked" but the interdependencies are tracked with these soft versions - there are lots of cases of flexible `^x.x.x` versions in the yarn.lock file already.  It looks like we now depend on two different versions of firmata:
```
firmata@^0.11.3:
  version "0.11.3"
  resolved "https://registry.yarnpkg.com/firmata/-/firmata-0.11.3.tgz#e122e4598aa1c67cd82165bf0fb727e5b89314f4"
  dependencies:
    browser-serialport "*"
    es6-shim "^0.33.6"
    serialport "^2.0.1"

firmata@latest:
  version "0.16.0"
  resolved "https://registry.yarnpkg.com/firmata/-/firmata-0.16.0.tgz#6f2a759b1a98de234ced010fa3d4d9534d62b2e7"
  dependencies:
    browser-serialport latest
    es6-shim latest
    serialport "^4.0.0"
```
The `^0.11.3` dependency is coming from `@code-dot-org/playground-io`.  In some cases, a single locked version is explicitly listed as fulfilling a whole set of flexible dependencies:
```
chalk@1.1.3, chalk@^1.0.0, chalk@^1.1.0, chalk@^1.1.1, chalk@^1.1.3, chalk@latest:
  version "1.1.3"
  resolved "https://registry.yarnpkg.com/chalk/-/chalk-1.1.3.tgz#a8115c55e4a702fe4d150abd3872822a7e09fc98"
  dependencies:
    ansi-styles "^2.2.1"
    escape-string-regexp "^1.0.2"
    has-ansi "^2.0.0"
    strip-ansi "^3.0.0"
    supports-color "^2.0.0"

chalk@^0.5.0, chalk@^0.5.1, chalk@~0.5.1:
  version "0.5.1"
  resolved "https://registry.yarnpkg.com/chalk/-/chalk-0.5.1.tgz#663b3a648b68b55d04690d49167aa837858f2174"
  dependencies:
    ansi-styles "^1.1.0"
    escape-string-regexp "^1.0.0"
    has-ansi "^0.1.0"
    strip-ansi "^0.3.0"
    supports-color "^0.2.0"

chalk@~0.4.0:
  version "0.4.0"
  resolved "https://registry.yarnpkg.com/chalk/-/chalk-0.4.0.tgz#5199a3ddcd0c1efe23bc08c1b027b06176e0c64f"
  dependencies:
    ansi-styles "~1.0.0"
    has-color "~0.1.0"
    strip-ansi "~0.1.0"
```

Anyway, it seems like I should update our playground-io fork to collapse our firmata dependency?